crates/flux-infer/src/fixpoint_encoding.rs

```diff
@@ -377,6 +377,7 @@ enum ConstKey<'tcx> {
     Alias(FluxDefId, rustc_middle::ty::GenericArgsRef<'tcx>),
     Lambda(Lambda),
     PrimOp(rty::BinOp),
+    ToInt(List<rty::SortArg>),
```

Comment:
```suggestion
    Cast(List<rty::SortArg>),
```

we could also make this `Cast(rty::Sort, rty::Sort)` to be very explicit.

Reply:
Ok got it. I think its fine to generate generic UIFs in fixpoint too... I don't think this matters too much one way or another. (The bigger issue is whether to use UIFs _at all_ in this case, or just disallow...)

Reply:
I was thinking that if we stop emitting generic UIFs that would make the transition to the fixpoint Rust reimplementation easier, but that's a long way ahead...